src/bgfx.cpp

```diff
@@ -1869,7 +1869,7 @@ namespace bgfx
 		frameNoRenderWait();
 
 		m_encoderHandle = bx::createHandleAlloc(g_allocator, _init.limits.maxEncoders);
-		m_encoder       = (EncoderImpl*)BX_ALLOC(g_allocator, sizeof(EncoderImpl)*_init.limits.maxEncoders);
+		m_encoder       = (EncoderImpl*)BX_ALIGNED_ALLOC(g_allocator, sizeof(EncoderImpl)*_init.limits.maxEncoders, 16);
```

Comment:
You must match freeing with BX_ALIGNED_FREE...

Reply:
https://github.com/bkaradzic/bgfx/pull/2188/commits/1dc0074cacef437a131ce810e646615ed7c15521